package.json

```diff
@@ -41,6 +41,7 @@
     "react-redux": "^8.1.3",
     "react-router": "^6.12.1",
     "react-router-dom": "^6.24.0",
+    "react-tooltip": "^5.28.0",
```

Comment:
We already have tool tip code.  Why is this new library needed?

Reply:
Used React component and css to create the banner since it is not depended on mouse action